google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseInfo.java

```diff
@@ -172,7 +199,8 @@ public Builder setProtoDescriptors(@Nonnull InputStream inputStream) throws IOEx
     @Override
     public Builder setProtoDescriptors(@Nonnull String filePath) throws IOException {
       Preconditions.checkNotNull(filePath);
-      Preconditions.checkState(filePath.length() != 0, "Input Proto Descriptors File Path cannot be empty.");
+      Preconditions.checkState(
+          filePath.length() != 0, "Input Proto Descriptors File Path cannot be empty.");
       InputStream inputStream = getClass().getClassLoader().getResourceAsStream(filePath);
```

Comment:
What will be the behaviour of this function if we give any random path? Can we mention it in function doc for clarity?

Reply:
The current functionality is to read file with in the classpath under `resources` directory. If the file path is random other than this it throws an `IOException: Input Proto Descriptors File path is invalid`.
I have added this behaviour in the `@throws` parameter of javadoc.